src/jit/compiler.hpp

```diff
@@ -4891,6 +4891,19 @@ void GenTree::VisitOperands(TVisitor visitor)
             return;
 #endif // FEATURE_SIMD
 
+#if FEATURE_HW_INTRINSICS
+        case GT_HWIntrinsic:
+            if ((this->AsHWIntrinsic()->gtOp1 != nullptr) && this->AsHWIntrinsic()->gtOp1->OperIsList())
```

Comment:
If we have added member to `GenTreeHWIntrinsic` which can be used to carry immediate values do we need `ArgList` for anything else than `Set` instructions?